frontend/test/jest/__tests__/components/UI/Table/EnvirTypeTable.spec.js

```diff
@@ -16,6 +16,7 @@ describe("Component: EnvirTypeTable", () => {
 			},
 		});
 	});
+
```

Comment:
Please move this modification in the "Add unit test spec". In doing this, this commit will be remove, if not remove it :)